nav2_map_server/test/unit/CMakeLists.txt

```diff
@@ -20,3 +18,11 @@ ament_target_dependencies(test_costmap_filter_info_server rclcpp)
 target_link_libraries(test_costmap_filter_info_server
   ${library_name}
 )
+
+target_link_libraries(test_map_io_2d
+  ${map_io_library_name}_2d
+  stdc++fs)
+
+target_link_libraries(test_map_io_3d
+  ${map_io_library_name}_3d
+  stdc++fs)
```

Comment:
I think, this block should be re-arranged to go after `test_map_io_*` target dependencies.